frontend/locale/de-DE.json

```diff
@@ -0,0 +1,5 @@
+{
+  "shareButton": {
```

Comment:
each extension should scope their own translations -->
`{
"pdpNativeShare" : {shareButton: {}}
}`

Reply:
done